node/Cargo.toml

```diff
@@ -36,6 +36,8 @@ jsonrpsee = { version = "0.16.2", features = ["server"] }
 pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.37" }
 sc-transaction-pool-api = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.37" }
 
+pallet-farming-rpc-api = { path = "../pallets/farming/rpc" }
+pallet-farming-rpc-runtime-api = { path = "../pallets/farming/rpc/runtime-api" }
```

Comment:
should log a task for integration tests of these new rpcs

Reply:
https://github.com/Manta-Network/Manta/issues/1183